### PR TITLE
stringify paginated results

### PR DESCRIPTION
**Pull request by shved**

### CODE REVIEW

**Comment by shved** [comment]
@taylorbrooks hey mate! looking for your attention <3

File: lib/closeio/client.rb [hunk 1]
```diff
@@ -65,7 +65,7 @@ def paginate(path, options={})
           skip += res['data'].count
         end
       end while res['has_more']
-      {has_more: false, total_results: res['total_results'], data: results.flatten}
+      {has_more: false, total_results: res['total_results'], data: results.flatten}.stringify_keys
```
Comment (taylorbrooks) [inline lib/closeio/client.rb]: `#stringify_keys` is not a Ruby method.

Explicitly set the string keys on the hash.

Comment (shved) [reply lib/closeio/client.rb]: @taylorbrooks OMG so shame! Thanks for the review. What do you think of using arrow hash notation here? 

https://github.com/taylorbrooks/closeio/pull/47/files#diff-161b0b1dfd24b2365f77b5cc2c8b9c1dR68

**Comment by shved** [comment]
@taylorbrooks I explicitly turned keys into strings. Lets merge it if it is good enough

**Comment by shved** [comment]
@taylorbrooks I'd close this PR for that I'm not going to finish this work and I event don't have a proper test account to try my changes and record VCRs with it. Hope somebody will do this changes.

**Pull request closed without merging**